components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/AuditConstants.java

```diff
@@ -36,6 +36,10 @@
     public static final String ACTION_PASSWORD_RECOVERY = "Password recovery";
     public static final String ACTION_USERNAME_RECOVERY = "Username recovery";
     public static final String ACTION_PASSWORD_RESET = "Password reset";
+    public static final String NOTIFICATION_TEMPLATE_TYPE = "Notification template";
```

Comment:
Can we use `IdentityRecoveryConstants.TEMPLATE_TYPE` instead of this constant?

Reply:
Those two have different values